<!DOCTYPE html>
<html lang="en">
   <head>
      <meta charset="utf-8">
      <title>Research</title>
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <meta name="description" content="Dr. Yang Liu's Research Lab, Home, University of Pittsburgh">
      <meta name="author" content="">
      <!-- Le styles -->
      <link href="css/bootstrap.min.css" rel="stylesheet">
      <link href="css/bootstrap-responsive.min.css" rel="stylesheet">
      <link href="css/theme.css" rel="stylesheet">
   </head>
   <body>
      <div class="container">
         <header class="jumbotron subhead" id="overview">
            <p class="lead"> University of Pittsburgh </p>
            <h1> Biomedical Optical Imaging Laboratory </h1>
         </header>
         <div class="masthead">
            <div class="navbar">
               <div class="navbar-inner">
                  <div class="container">
                     <ul class="nav">
                        <li><a href="index.html">Home</a></li>
                        <li><a href="people.html">People</a></li>
                        <li class="active"><a href="#">Research</a></li>
                        <li><a href="publications.html">Publications</a></li>
                        <li><a href="gallery.html">Gallery</a></li>
                        <li><a href="software.html">Software</a></li>
                        <li><a href="contact.html">Openings</a></li>
                     </ul>
                  </div>
               </div>
            </div>
         </div>
         <hr>
         <div class="row-fluid">
            <div class="span3 bs-docs-sidebar" id="navparent">
               <ul class="nav nav-list bs-docs-sidenav" data-spy="affix" data-offset-top="200" data-offset-bottom="260">
                  <li><a href="#statemet"> Overview </a></li>
                  <li><a href="#nano"> Technology Development </a></li>
                  <li><a class="subhead" href="#STORM"> High-throughput multi-color STORM </a></li>
                  <li><a class="subhead" href="#miniSTORM"> miniSTORM </a></li>
                  <li><a class="subhead" href="#nanoNAM"> nanoNAM and Quantitative Phase Imaging </a></li>
                  <li><a class="subhead" href="#Sheet"> Light-sheet Microscopy </a></li>
                  <li><a href="#Facilisis">Biomedical Applications </a></li>
                  <li><a class="subhead" href="#Cancer"> Cancer Detection </a></li>
                  <li><a class="subhead" href="#Chromatin"> Chromatin Biology </a></li>
                  <li><a class="subhead" href="#Aging"> Aging </a></li>
               </ul>
            </div>
            <div class="span8 offset1">
               <section id="statemet">
                  <div class="page-header">
                     <h3>Overview</h3>
                     <hr>
                     <p> The laboratory of Dr. Yang Liu integrates a multidisciplinary approaches of engineering, chemistry, physics, computer science, medicine and biology and develop advanced optical 
                        imaging technologies and biosensors to improve cancer detection and treatment. We exploit the power of light and develop a suite of nanoscale imaging techniques and sensors. 
                        We push the limit of imaging technologies  the technical capability by making them faster, cheaper and more robust. We apply these tools to detect subtle molecular changes in 
                        cells at risk for cancer development and cancer biomarkers at single molecule sensitivity. We also use these powerful imaging tools to advance our understanding of cancer 
                        biology and learn how normal cells are transformed into malignancy and why their natural courses differ in different patients. Our ultimate goal is to personalize cancer detection, 
                        risk stratification and treatment.
                     </p>
                  </div>
               </section>
               <section id="nano">
                  <div class="page-header">
                     <h3>Technology Development</h3>
                  </div>
                  <div class="row-fluid">
                     <div class="span12">
                        <section id="STORM">
                           <h4>High-throughput STORM</h4>
                           <br/>
                           <img src="images/HT_STORM-2.jpg" class="float-right" alt="STORM"/>
                           <p>
                              Advances in super-resolution fluorescence microscopy have revolutionized biological imaging by overcoming the fundamental diffraction barrier, recognized by a Nobel Prize in 2014. 
                              In particular, single molecule localization microscopy (SMLM) such as (fluorescence) photo-activated localization microscopy [(f)PALM] and (direct) stochastic optical reconstruction 
                              microscopy [(d)STORM] is one of the simplest and most cost-effective approaches to achieve super-resolved imaging capability at a spatial resolution down to 5-10 nm. 
                              Despite its impact, it is generally deemed as a complex and low-throughput technique. The main rate-limiting factors are the small illumination field, the need to acquire 
                              tens of thousands of imaging frames to reconstruct an image and slow image reconstruction speed. It is also limited to imaging thin and transparent samples (bacteria, cultured 
                              cells). We aim to develop multi-color high-throughput STORM system that can be used to analyze clinically relevant pathological samples and apply machine-learning tools for 
                              automated analysis. 
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              1. [Xu J#, Ma HQ#], Ma HB, Jiang W, Duan M, Mela CA, Zhao S, Gao C, Hahm E-R, Lardo SM, Troy K, Sun M, Pai R, Stolz DB, Zhang L, Singh S, Brand RE, 
                              Hartman DJ, Hu J, Hainer SJ, Liu Y. Super-resolution imaging reveals the evolution of higher-order chromatin folding in early carcinogenesis. <a href="https://www.biorxiv.org/content/10.1101/672105v1"> 
                              Nature Communications, in press, 2020. </a>
                              <br>
                              2. Ma H, Xu J, Liu Y. WindSTORM: Robust online image processing for high-throughput nanoscopy.<a href="https://advances.sciencemag.org/content/5/4/eaaw0683">
                                 Science Advances, eaaw0683, 2019.</a> 
                            </p>
                        </section>
                        <hr>
                        <section id="miniSTORM">
                           <h4>miniSTORM</h4>
                           <br/>
                           <img src="images/miniSTORM_figure-1.jpg" class="float-right" alt="miniSTORM"/>
                           <p>
                              Although super-resolution microscopy is commercially available, it remains a high-end microscopy instrument only available in the core imaging centers at major academic institutions 
                              and a small number of well-funded laboratories around the world. The high cost of this technology limits its widespread use as a routine microscopy system. Small colleges and 
                              commercial laboratories often cannot afford such expensive instruments. We aim to develop miniSTORM, an affordable state-of-the-art nanoscopy with a spatial resolution one 
                              order-of-magnitude better than conventional light microscopy. The hardware cost will be 10 times lower than commercial systems. It is compact and can be put on a regular table 
                              top without a dedicated room and high-end optical table. It is highly robust and automated. Importantly, miniSTORM integrates artifact minimization modules that provide 
                              high-quality image, it will reduce the learning curve and the need for significant user expertise. 
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              1. Ma H, Fu R. Xu J, Liu Y. A simple and cost-effective setup for super-resolution localization microscopy.<a href="https://www.nature.com/articles/s41598-017-01606-6"> Scientific Reports, 7(1): 1542, 2017.</a>
                              <br>
                              2. Ma H, Xu J, Jin J, Huang Y, Liu Y. A simple marker-assisted three-dimensional nanometer drift correction method for super-resolution microscopy.<a href="https://www.cell.com/biophysj/fulltext/S0006-3495(17)30441-1"> 
                                 Biophysical Journal, 112: 2196-2208, 2017. </a>
                           </p>
                        </section>
                        <hr>
                        <section id="nanoNAM">
                           <h4>nanoNAM and Quantitative Phase Imaging</h4>
                           <br/>
                           <img src="images/nanoNAM-1.jpg" class="float-right" alt="Ullamcorper"/>
                           <p>
                              Nanoscale nuclear architecture mapping (nanoNAM) is an  interferometry approach that builds on the well-established principle behind Spectral-domain 
                              Optical Coherence Tomography (SD-OCT) to allow low-amplitude back-scattered light to “ride” the reference signal with shared optical path. These 
                              back scattered signals capture the sub-resolution nanoscale structural alterations within the cell nucleus with slowly varying refractive index 
                              without a strong interface. Based on our Fourier mapping approach, nanoNAM computes the Fourier phase associated with these back-scattered waves 
                              to capture, with nanoscale sensitivity, (a) depth-resolved alterations in mean optical density, and (b) depth-resolved localized heterogeneity 
                              in optical density of the cell nuclei. This depth-resolved capability of nanoNAM allows 3D quantification of nuclear architecture, while 
                              simultaneously avoiding the confounding effect of thickness variation. We apply nanoNAM together with quantitative phase imaging to explore its ability
                              to detect subtle structural changes in refractive index in cells undergoing early-stage carcinogenesis.
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              1. [Uttam S*, Pham HV*], LaFace J, Leibowitz B, Yu J, Brand RE, Hartman DJ, Liu Y. Early prediction of cancer progression by depth-resolved nanoscale maps of nuclear architecture from unstained tissue specimens.
                                 <a href="https://cancerres.aacrjournals.org/content/75/22/4718.long/"> Cancer Research, 75(22): 4718-4727, 2015.</a>
                              <br>
                              2. Uttam S, Hashash JG, LaFace J, Binion D, Regueiro M, Hartman DJ, Brand RE, Liu Y. Three-Dimensional Nanoscale Nuclear Architecture Mapping of Rectal Biopsies Detects Colorectal Neoplasia in Patients with Inflammatory Bowel Disease.
                              <a href="https://cancerpreventionresearch.aacrjournals.org/content/12/8/527">Cancer Prevention Resesarch, 12(8), 527-538, 2019.</a>
                              <br>
                              3. Uttam S, Liu Y. Fourier phase based depth-resolved nanoscale nuclear architecture mapping for cancer detection.
                              <a href="https://www.sciencedirect.com/science/article/pii/S1046202317303146?via%3Dihub"> Methods, 136: 134-151, 2018.</a> 
                              <br>
                              4. Uttam S, Liu Y. Fourier phase in Fourier-domain optical coherence tomography. 
                              <a href="https://www.osapublishing.org/josaa/abstract.cfm?uri=josaa-32-12-2286"> Journal of the Optical Society of America A, 32(12):2286-2306, 2015.</a>
                              <br>
                              5. Pham HV, Pantanowitz L, Liu Y. Quantitative phase imaging to improve the diagnostic accuracy of urine cytology. 
                              <a href="https://acsjournals.onlinelibrary.wiley.com/doi/full/10.1002/cncy.21734"> Cancer Cytopathology, 124:641-650, 2016. </a>
                           </p>
                        </section>
                        <hr>
                        <section id="Sheet">
                           <h4>Light-sheet Microscopy</h4>
                           <br/>
                           <img src="images/Spim.jpg" class="float-right" alt="SPIM"/>
                           <p>
                              Light-sheet microscopy is a powerful approach to minimize photobleaching effect and reduce the out-of-focus background. However, the physical constraint of the objective limits
                              the use of high-NA optics, thus limiting the optical resolution. Significant progress has been made in the development of high-resolution light-sheet systems. Our group aims to 
                              develop simple high-resolution light-sheet microscopy systems for 3D live-cell imaging for various biomedical applications.
                           </p>
                        </section>
                     </div>
                  </div>
               </section>
               <hr>
               <section id="Facilisis">
                  <div class="page-header">
                     <h3>Biomedical Applications</h3>
                  </div>
                  <div class="row-fluid">
                     <div class="span12">
                        <section id="Cancer">
                           <h4>Cancer Detection</h4>
                           <br/>
                              <img src="images/STORM_cancer.jpg" class="float-right" alt="Cancer"/>
                           <p>
                              Cancer risk stratification—identifying subjects at high risk for developing cancer—is an important tool 
                              in cancer prevention and early detection, and crucial for reducing cancer-related morbidity and mortality. 
                              However, due to the immense molecular heterogeneity at each stage of carcinogenesis, few tools are available 
                              to accurately identify the small percentage of subjects at the highest risk for developing cancer and 
                              who would benefit most from early detection and cancer prevention strategies.
                           </p>
                           <p>
                              Abnormal chromatin structure is one of the most universal and striking characteristics in cancer cells and 
                              has remained the gold standard for cancer diagnosis for two centuries, but it often lacks sufficient 
                              sensitivity to accurately identify patients at highest risk for developing cancer. As a result, under current technology, 
                              cells undergoing early carcinogenesis may not appear pathologically distinct. That is, higher-risk pre-cancerous 
                              lesions destined to undergo progression to cancer may be indistinguishable from low-risk precursors. Uncertain cancer 
                              risk can lead to sub-optimal clinical management of at-risk patients: high-risk patients may undergo inadequate 
                              surveillance, whereas the lower-risk subjects may undergo excessive testing or unnecessary interventions. 
                           </p>
                           <p>
                              We aim to bring nanoscale resolution from super-resolution microscopy into prognostication of subsequent cancer risk, by 
                              detecting the cumulative disruption in chromatin folding at nanoscale to characterize the advancement of a given precursor. 
                              We are exploring this technique in the cancer risk stratification of colon polyps, and early detection of lung cancer.   
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              [Xu J#, Ma HQ#], Ma HB, Jiang W, Duan M, Mela CA, Zhao S, Gao C, Hahm E-R, Lardo SM, Troy K, Sun M, Pai R, Stolz DB, Zhang L, Singh S, Brand RE, 
                              Hartman DJ, Hu J, Hainer SJ, Liu Y. Super-resolution imaging reveals the evolution of higher-order chromatin folding in early carcinogenesis. <a href="https://www.biorxiv.org/content/10.1101/672105v1"> 
                              Nature Communications, in press, 2020. </a>
                              <br> 
                            </p>
                        </section>
                        <hr>
                        <section id="Chromatin">
                           <h4>Chromatin Biology</h4>
                           <br/>
                           <img src="images/CellReports.jpg" class="float-right" alt="Chromatin"/>
                           <p>
                              Eukaryotic cells package genomic DNA up to 2 meters long into a nucleus with a diameter of several microns through a hierarchical 
                              scheme of compaction into DNA-protein assemblies. The first level is nucleosome, consisting of 147 base pairs of DNA wrapped 
                              around an octamer of four core histone proteins. This basic repeating unit of nucleosomes is then organized into ~10 nm “beads-on-string” 
                              chromatin fiber, which is further compacted into higher-order chromatin structure, to fit into the micron-sized nucleus. Chromatin 
                              organization is regulated by a large number of chemical modifications, particularly on the N-terminal tails of histone core proteins, 
                              such as acetylation and methylation. Histone modifications regulate the packaging of nucleosomes into higher-order chromatin structure 
                              to influence the accessibility of genomic DNA to the transcription machinery proteins. Subsequently, chromatin compaction at different 
                              epigenomic states controls their gene expression and imposes a significant effect on many cellular processes, such as DNA replication, 
                              cell division, DNA damage and DNA repair. We apply powerful super-resolution microscopy and genomic technologies (ChIP-seq and RNA-seq) to 
                              study higher-order chromatin organization at differnet epigenomic states and aim to understand how chromatin organization coordinates 
                              with co-factors to impact gene expression in normal cellular processes and pathological states.
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              Xu J, Ma H, Jin J, Uttam S, Fu R, Huang Y, Liu Y. Super-resolution imaging of higher-order chromatin structures at different epigenomic states in single mammalian cells. 
                              <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(18)31012-X/"> Cell Reports, 24: 873-882, 2018. </a>  
                           </p>
                        </section>
                        <hr>
                        <section id="Aging">
                           <h4>Aging</h4>
                           <br/>
                           <img src="images/nuclearlamina-1.jpg" class="float-right" alt="Aging"/>
                           <p>
                              The nuclear lamina is an intermediate filament meshwork adjacent to the inner nuclear membrane that plays a critical role in maintaining nuclear shape and regulating 
                              gene expression through chromatin interactions. In collaboration with Dr. Quasar Padiath from Department of Human Genetics at University of Pittsburgh, Dr. Kris Dahl from 
                              Carnegie Mellon University and Dr. Bernhard Kuhn from Chilren's Hospital of Pittsburgh, we apply multi-color super-resolution microscopy to characterize distinct ultra-structures of nuclear lamina proteins
                              and understand their functional roles in normal cellular processes and aging. 
                           </p>
                           <p>
                              <b>Relevant Publcations: </b> <br>
                              [Nmezi B#, Xu J#, Fu R#], Armiger T, Bey G, Schneider J, Ma H, Tu Y, Chen N, Young S, [Dahl K*, Liu Y*, Padiath Q*]. Concentric organization of A- and B-type lamins predicts their distinct roles in the spatial organization and stability of the nuclear lamina. 
                              <a href="https://www.pnas.org/content/116/10/4307/"> Proceedings of the National Academy of Sciences of the USA, 116(10): 4307-4315, 2019. </a>
                           </p>
                        </section>
                     </div>
                  </div>
               </section>
               <hr>
            </div>
         </div>
      </div>
      <footer id="footer">
         <div class="container-fluid">
            <div class="row-fluid">
               <div class="span5">
                  <h3>Contact Information</h3>
                    <p><b>Phone: </b>1-412-6233751</p>
                   <a href="mailto:liuy@pitt.edu">Email</a>
               </div>
               <div class="span2">
                  <a href="https://profiles.dom.pitt.edu/gi/faculty_info.aspx/Liu5712/"><img src = "thumbs/YangLiuLabMembers.jpg" alt="research-lab-logo"/></a>
               </div>
               <div class="span5">
                  <h3>Address</h3>
                  <p>Hillman Cancer Center 2.32e <br>
                     5117 Centre Avenue<br>
                     Pittsburgh, PA <br>
                     USA 15232
                  </p>
                  <a href="https://goo.gl/maps/rGxg36Jwa1Y647JC6">Show Map</a>
               </div>
            </div>
         </div>
      </footer>

      <!-- Le javascript
         ================================================== -->
      <!-- Placed at the end of the document so the pages load faster -->
      <script src="js/jquery-1.9.1.min.js"></script>
      <script src="js/bootstrap.min.js"></script>
      <script>
         $(document).ready(function() {
             $(document.body).scrollspy({
                 target: "#navparent"
             });
         });
         
      </script>
   </body>
</html>